src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/src/read_ir/read_ir_compare_with_refs.cpp

```diff
@@ -16,13 +16,27 @@ using namespace ov::test::subgraph;
 
 namespace {
 
-INSTANTIATE_TEST_SUITE_P(conformance,
-                         ReadIRTest,
-                         ::testing::Combine(
-                                 ::testing::ValuesIn(getModelPaths(IRFolderPaths)),
-                                 ::testing::Values(targetDevice),
-                                 ::testing::Values(pluginConfig)),
-                         ReadIRTest::getTestCaseName);
+#include <ngraph/ngraph.hpp>
+
+#define NGRAPH_OP(NAME, NAMESPACE)                                                                         \
+    INSTANTIATE_TEST_SUITE_P(conformance##NAME,                                                            \
+                             ReadIRTest,                                                                   \
+                             ::testing::Combine(::testing::ValuesIn(getModelPaths(IRFolderPaths, #NAME)),  \
+                                                ::testing::Values(targetDevice),                           \
+                                                ::testing::Values(pluginConfig)),                          \
+                             ReadIRTest::getTestCaseName); \
+
+// It should point on latest opset which contains biggest list of operations
+#include <ngraph/opsets/opset10_tbl.hpp>
```

Comment:
we will lost other opsets. Need think how to specify for other opsets. Command-line?

Reply:
Nop, We use only operation-names without opsets. So, it will verifiy all opsets of each operation.

Biggest issue which we would face with such approach - if some operation will be deleted in some future opset (for example, if Add will be fully removed from a list).